code/game/atoms.dm

```diff
@@ -76,7 +76,7 @@
 	/// A luminescence-shifted value of the last color calculated for chatmessage overlays
 	var/chat_color_darkened
 	///HUD images that this mob can provide.
-	var/list/hud_possible
+	var/list/hud_possible = list(XENO_TACTICAL_HUD)
```

Comment:
why is this on atom

Reply:
Future proofing; I'm changing Rally Hive to atom targeting per Tivi's request

Reply:
this adds a new list to **every single atom in the world**

Reply:
^

Reply:
This should be ready to go.